omou-frontend/src/components/Form/Form.js

```diff
@@ -307,24 +308,24 @@ class Form extends Component {
                     };
                     console.log(NewState[label]);
```

Comment:
this may have been leftover from something we did before. can you delete this?

Reply:
Got it